pkg/test/case_test.go

```diff
@@ -20,7 +20,7 @@ func TestLoadTestSteps(t *testing.T) {
 		testSteps []Step
 	}{
 		{
-			"test_data/with-overrides/",
+			"../../test/integration/with-overrides/",
```

Comment:
I strongly support moving testing to the `test` folder from root.   This is my least favorite part of it.  "../../test".  so every test we will need to know how deep we are in the package structure and every time we refactor packages we will need to deal with this?   It seems easy for a human to fat finger.   Is there anything we can think of to solve this in a better way?

Reply:
This is just one or two cases that exercise various parts of the test suite.

So, two ideas:

* We leave it with the `../../test/integration/with-overrides/`. This would only really break if someone changed the `with-overrides` test for some reason.
* We leave the tests that `case_test.go` inspects in `pkg/test/test_data` since they are literally that, just test data.

Reply:
I think I'll go with option two. Especially since they aren't testing KUDO, just the harness.

Reply:
Fixed!